.jenkins/Dockerfile

```diff
@@ -5,6 +5,7 @@ LABEL description="Automated HomeStore compilation"
 WORKDIR /output
 
 RUN set -eux; \
+    sed -i 's/master/latest/g' /etc/apt/sources.list; \
```

Comment:
Is this change needed in develop?

Reply:
I think so, otherwise when building you can still run into the issue when running apt-get update

Reply:
every branch is failing without this line in image state including AM